### ISSUE TITLE
Suppressed unchecked exceptions highlighted in IntelliJ for 3rd party dependencies

### ISSUE THREAD

**Issue by elshimone** [manifold-systems/manifold#74]
**Describe the bug**
I think there may be a minor issue with the IntelliJ plugin still reporting unchecked exception errors for third party dependencies. The project still builds as expected, the issue is limited to the IntelliJ plugin support.

**To Reproduce**
Modify the UserClient class in the Manifold REST API (https://github.com/manifold-systems/manifold-sample-rest-api) project to:

```java
public class UserClient {
  public static void main(String[] args) {

    // suppressed
    new Dummy().exception();

    // suppressed
    new URL("http://manifold.systems");

    XmlParser pa = new XmlParser();
    // Displays error for unhandled exceptions IOException, SAXException
    pa.parse("<xml/>");
  }
}
class Dummy {
  void exception() throws IOException {
    throw new IOException("yikes");
  }
}

```

**Expected behavior**
I would expect the errors to be supressed in all cases.

**Screenshots**
![image](https://user-images.githubusercontent.com/698207/57195178-96826580-6f47-11e9-9cc7-7bded14f8b5c.png)

**Desktop (please complete the following information):**
- OS Type & Version: Mac OSX 10.13.6 (17G6030)
- Java/JDK version: Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)
- IntelliJ IDEA version: IntelliJ IDEA 2019.1.1 (Community Edition) Build #IC-191.6707.61, built on April 16, 2019
- Manifold version: 0.63-alpha
- Manifold IntelliJ plugin version: 0.63-alpha

**Additional context**
None

**Stack trace**
None

**Comment by RaviH** [comment]
+1

**Comment by wedsa5** [comment]
+1

**Comment by rsmckinney** [comment]
In progress

**Comment by rsmckinney** [comment]
Fix available in release 0.65-alpha
